settings.gradle.kts

```diff
@@ -99,6 +99,7 @@ include("persistentCache")
 include("buildCache")
 include("coreApi")
 include("versionControl")
+include("coreFileServices")
```

Comment:
I wonder if we should drop "core" from this and just call it "files" or "file-operations"?

Reply:
As I went ahead that `core` prefix felt more and more out-of-place to me, too. I'm not sure about `operations` – the `FileOperations` interface decidedly stayed in `core` as it is more about file manipulation. This one is more about _modeling_ files and collections of files. So, maybe `file-model`?

Reply:
Every subproject should model some domain concept, right? I don't see why we would add `model` to the artifact name. `file` sounds good to me and captures the intent well I think.